File: clients/typescript/examples/health_smoke.ts
/** W40-05: TypeScript client smoke example. */
const addr = process.env.KNXVAULT_ADDR ?? 'http://127.0.0.1:8200';

async function main(): Promise<void> {
  const res = await fetch(`${addr}/health`);
  console.log(await res.text());
}

main().catch(console.error);

File: clients/rust/examples/health_smoke.rs
//! W40-07: Rust client smoke example.
fn main() {
    let addr = std::env::var("KNXVAULT_ADDR").unwrap_or_else(|_| "http://127.0.0.1:8200".into());
    let url = format!("{addr}/health");
    let body = reqwest::blocking::get(&url)
        .expect("health request")
        .text()
        .expect("body");
    println!("{body}");
}